manifests/process_exporter.pp

```diff
@@ -43,6 +43,27 @@
 #  User which runs the service
 # @param version
 #  The binary release version
+# @param hash_watched_processes
+#  A hash of processes to monitor with the ability to pass different options
+#  Don't set if you want to use only the Array version of it (watched_processes)
+#  Usage example:
```

Comment:
generally, we'd use `@example Usage` for the entire class, see [documentation and examples](https://puppet.com/docs/puppet/6.10/puppet_strings_style.html)

Reply:
btw, that doesn't mean you have to remove it, just move it a bit further down and convert use the `@example` Notation.